### PR TITLE
Workshopmodel

### PR DESCRIPTION
**Pull request by JTBreunissen**
Created model Workshop and added validation tests

### CODE REVIEW

**Comment by arnoFleming** [comment]
please merge master into this branch, and run migrate your database to get rid of the conflict in `db/schema.rb`.

**Comment by JTBreunissen** [comment]
I merged master into branch, ran the migration and pushed the branch again, it did however also include the work I was doing in a different branch. Specifically the addition of the footer to the application.html.erb.

File: spec/models/workshop_spec.rb [hunk 1]
```diff
@@ -3,21 +3,27 @@
 RSpec.describe Workshop, type: :model do
   describe "validations" do
     it "is invalid without a title" do
-      workshop = Workshop.new(title: "")
+      workshop = FactoryGirl.build(:workshop, title: "")
```
Comment (arnoFleming) [inline spec/models/workshop_spec.rb]: if you added Factory to your spec helper, you could (just) write  `workshop = build(:workshop, title: "")`

File: spec/factories/workshops.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :workshop do
+    title "MyString"
```
Comment (arnoFleming) [inline spec/factories/workshops.rb]: maybe update the contents of this file so it makes sense to human beings

**Pull request merged**